documentation/src/main/asciidoc/topics/con_xsite_backup_strategies.adoc

```diff
@@ -1,5 +1,41 @@
-[id='xsite_backup_strategies-{context}']
+[id='backup-strategies_{context}']
 = Backup Strategies
-{brandname} clusters can use different strategies for backing up data to remote sites.
 
-{brandname} replicates across sites at the same time that writes to local caches occur. For example, if a client writes "k1" to **LON**, {brandname} backs up "k1" to **NYC** at the same time.
+{brandname} replicates data between clusters at the same time that writes to caches occur.
+For example, if a client writes "k1" to **LON**, {brandname} backs up "k1" to **NYC** at the same time.
+
+To back up data to a different cluster, {brandname} can use either a synchronous or asynchronous strategy.
+
+[discrete]
+== Synchronous Backups
+When {brandname} replicates data to backup locations, it waits until the operation completes before writing to the local cache.
```

Comment:
not true :) the write to the local cache happens concurrently with the remote site.  
Infinispan waits for both the local and the remote site writes before returning the operation